src/extensions/default/OpenWithExternalApplication/main.js

```diff
@@ -49,7 +49,8 @@ define(function (require, exports, module) {
     var extensionToExtApplicationMap = {};
 
     function convertUnixPathToWindowsPath(path) {
-        if (brackets.platform === "win" && path && path[1] === ":" && path[2] === "/") {
+        if (brackets.platform === "win"
```

Comment:
What is the reason for this change? Windows network paths?

Reply:
when we open network file with some external application, it does not accept path with forward-slash "/", so need to converted to windows style.

e.g

Illustrator //<network_path_with_forward_slash>/example.ai    this does not work

so have to give like : Illustrator \\<network_path_with_backward_slash>\example.ai

Reply:
Can't we use the condition  `brackets.platform === "win" && FileSystem.isAbsolutePath()`.

Reply:
Also are we planning to remove leading slashes from paths like `/C:/notepad/notepad.exe`?
And more than two leading slashes do not work for network path, so we are covered there.

Reply:
Here we will never have local path with leading slashes so not required.